arangod/Cluster/ClusterFeature.cpp

```diff
@@ -299,20 +312,8 @@ void ClusterFeature::validateOptions(std::shared_ptr<ProgramOptions> options) {
     FATAL_ERROR_EXIT();
   }
 
-  // validate
-  if (_agencyPrefix.empty()) {
-    _agencyPrefix = "arango";
-  }
-
-  // validate --cluster.agency-prefix
-  size_t found = _agencyPrefix.find_first_not_of(
-      "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789/");
-
-  if (found != std::string::npos || _agencyPrefix.empty()) {
-    LOG_TOPIC("7259b", FATAL, arangodb::Logger::CLUSTER)
-        << "invalid value specified for --cluster.agency-prefix";
-    FATAL_ERROR_EXIT();
-  }
+  // changing agency namespace no longer needed
+  _agencyPrefix = "arango";
```

Comment:
do we need to support cases in which end users have adjusted the agency prefix themselves?
is this a concern?

Reply:
No, this does not even work, because other places have "arango" hardcoded. I marked this options as obsolete.

Reply:
Fine, then we don't need to support it, but just mention it in the CHANGELOG plus the "upgrading-changes" document in the docs repository.